plugins/woocommerce/client/admin/client/customize-store/assembler-hub/opt-in/opt-in.tsx

```diff
@@ -86,19 +86,18 @@ async function installFonts(
 		...( custom ? custom.map( ( font ) => font.slug ) : [] ),
 	];
 
-	const fontFamiliesToEnable = parsedInstalledFontFamilies.reduce(
-		( acc, font ) => {
-			if (
-				enabledFontSlugs.includes( font.slug ) ||
-				FONT_FAMILIES_TO_INSTALL[ font.slug ] === undefined
-			) {
-				return acc;
-			}
-
-			return [ ...acc, { ...font } ];
-		},
-		[] as Array< FontFamily >
-	);
+	const fontFamiliesToEnable = parsedInstalledFontFamilies
```

Comment:
This change does not improve the codebase. In fact, it does not offer better performance compared to the `reduce` implementation.

Analyzing the runtime complexity, this code executes at:
`O(N*M) + O(N)`
where `N = parsedInstalledFontFamilies.length` and `M = enabledFontSlugs.length`.

In contrast, the `reduce` version operates at `O(N*M)`, which is effectively `O(N)` faster than the `map` implementation.

An improvement to this implementation could involve converting `enabledFontSlugs` into a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) of this form `{[fontSlug: string]: boolean}`. Doing so would reduce the runtime complexity of the `reduce` version to `O(N)`, as the need to loop through `enabledFontSlugs` is eliminated. With a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), the lookup operation is amortized to `O(1)`.